.github/workflows/test-digital-ocean-1-click.yaml

```diff
@@ -1,10 +1,18 @@
-name: e2e - DigitalOcean 1-click (install)
-
-on: push
+name: e2e - DigitalOcean (1-click)
+#
+# Unfortunately we can test this scenario only after merging to 'main' as we don't have
```

Comment:
I've realised about this limitation while working on the non 1-click install. I'll try to come up with a workaround as this situation is not ideal.

Reply:
briefly discussed in our 1:1 that we could use an env variable that's read in `deploy.sh` & define it after we create the DO cluster, then run the deployment script